music21/articulations.py

```diff
@@ -596,27 +595,37 @@ class FretBend(FretIndication):
     Release is the quarterLength value from the start
     of the note for releasing the bend, if Any. Defaults to None.
 
-    WithBar indicates if the bend is done using a whammy bar movement. Defaults to False.
+    WithBar indicates what whammy bar movement is used, if any.
```

Comment:
If possible write camelCase variables w/ proper camelCase even at start of the line, using backticks `withBar` can make it look fine in the docs even at the start of the sentence.  Sorry I didn't get this last time.